packages/client/src/comp/About.tsx

```diff
@@ -1,39 +1,103 @@
 import React from 'react'
-import { processEnvOrThrow } from '../common'
-import { useAppHistory } from '../lib/path'
-import { RoundedButton } from './util/Button'
-import { SocialFollow } from './SocialFollow'
-import { SocialShare, SocialButtonWrapper } from './SocialShare'
-import { Row, Col } from 'muicss/react'
-
-export const About = () => {
-  const brandName = processEnvOrThrow('REACT_APP_BRAND_NAME')
-  const url = processEnvOrThrow('REACT_APP_URL')
-  const { pushStartSection } = useAppHistory()
-
-  return <div>
-    <h1>About Us</h1>
-    <p>
-      COVID-19 has catalyzed interest in Vote by Mail (VBM).  <a href={url}>{brandName}</a> streamlines government VBM applications by digitizing the voter’s VBM signup process.  The mobile-friendly website drives voter turnout in key swing states by helping organizers sign up voters for VBM.
-    </p>
-    <SocialButtonWrapper>
-      <RoundedButton
-        color='primary'
-        variant='raised'
-        onClick={() => pushStartSection('start')}
-      >
-        <span>Get Started</span>
-      </RoundedButton>
-    </SocialButtonWrapper>
-    <Row>
-      <Col md={6} style={{ textAlign: 'center' }}>
-        <h3>Follow</h3>
-        <SocialFollow/>
-      </Col>
-      <Col md={6} style={{ textAlign: 'center' }}>
-        <h3>Share</h3>
-        <SocialShare/>
-      </Col>
-    </Row>
-  </div>
+import styled from 'styled-components'
+
+import img1 from './img/about_pic1.jpg'
+import img2 from './img/about_pic2.jpeg'
+import { cssQuery } from './util/cssQuery'
+import { MarketingWrapper } from './util/MarketingWrapper'
+import { Container } from 'muicss/react'
+
+const Wrapper = styled(MarketingWrapper)`
+  background-color: #fafafa;
+  box-shadow: 0 12px 14px -10px rgba(0, 0, 0, 0.25) inset;
+
+  .mui-container {
+    flex-flow: column;
+    ${cssQuery.mobile.narrow} {
+      padding-left: 0;
+      padding-right: 0;
+    }
+
+    ${cssQuery.desktop.all} {
+      flex-direction: row-reverse;
+      align-items: flex-start;
+      flex-flow: row-reverse wrap;
+    }
+  }
+
+`
+
+const Headline = styled.div`
+  width: 90%;
+  text-align: left;
+  h1 { padding-top: 0; }
+  ${cssQuery.mobile.narrow} { h1 { padding-top: 0; } }
+
+  ${cssQuery.desktop.all} { width: 40%; p { width: 80%; } }
+  ${cssQuery.mobile.wide} { width: 50%; }
+`
+
+// The images use vh as its sizing factor to ensure all devices can get the
+// same overlapped effect.
+const Images = styled.div`
+  --imageHeight: 250px;
+
+  position: relative;
+  width: 100%;
+  height: calc(var(--imageHeight) * 2);
+  margin-bottom: 50px;
+  ${cssQuery.mobile.wide} { width: 70%; }
+  ${cssQuery.desktop.all} {
+    width: 35%;
+    height: calc(var(--imageHeight) * 1.8);
+    margin-bottom: 0;
+  }
+
+  img {
+    max-width: 90%;
+    height: var(--imageHeight);
+
+    object-fit: cover;
+    position: absolute;
+
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.25);
+
+    &:nth-child(1) {
```

Comment:
I would call this `&.upper-left` and `&.bottom-right` or whatever the correct descriptive thing is.